values-schema.yaml

```diff
@@ -186,6 +186,9 @@ definitions:
             type: string
             description: Azure Application Client Secret
             x-secret: ''
+          secretName:
+            type: string
+            description: Existing secret with credentials. Will override credentials provided.
```

Comment:
Could you explain in `adr` when and where a user defines the secret.
Please let us know about user story related with using that secret.

Reply:
Its documented in external-dns chart and is always the same in any chart: provide a secret name instead of credentials. No further explanation necessary imo.

Reply:
ok, but does it mean a user needs to create that secret before installing Otomi? If so, does a user need to create `external-dns` namespace ?

Reply:
indeed...same as with other charts that rely on pre-created secrets. I could add a little section that says "Expects the secret to exist in this app's namespace."

Reply:
but we are not in that context when defining the schema, so I'd leave it as is

Reply:
But I see a bigger issue where we have to get to the secrets from external-dns from the cert-manager namespace. So this involves copying? Hmmmm. We're not done yet

Reply:
So do we want to support that case now?